echopype/utils/align.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import xarray as xr
+
+
+def align_to_ping_time(
+    external_da: xr.DataArray,
+    external_time_name: str,
+    ping_time_da: xr.DataArray,
+    method: str = "nearest",
+) -> xr.DataArray:
+    """
+    Aligns an external DataArray to align time-wise with the Echosounder ping time DataArray.
+
+    Parameters
+    ----------
+    external_da : xr.DataArray
+        External Non-Echosounder data.
+    external_time_name : str
+        Time variable name of the External Non-Echosounder data.
+    ping_time_da : xr.DataArray
+        Echosounder ping time.
+    method : str, default 'nearest'
+        Interpolation method. Not used if external time matches ping time or external
+        DataArray is a single value.
+        For more interpolation methods please visit: https://docs.xarray.dev/en/stable/generated/xarray.DataArray.interp.html # noqa
```

Comment:
Just removing some capitalizations
```suggestion
    Aligns an external DataArray to align time-wise with the echosounder ping time DataArray.

    Parameters
    ----------
    external_da : xr.DataArray
        External non-echosounder data.
    external_time_name : str
        Time variable name of the external non-echosounder data.
    ping_time_da : xr.DataArray
        Echosounder ping time.
    method : str, default 'nearest'
        Interpolation method. Not used if external time matches ping time or external
        DataArray is a single value.
        For more interpolation methods please visit: https://docs.xarray.dev/en/stable/generated/xarray.DataArray.interp.html # noqa
```